### ISSUE TITLE
Camera produce !is_inside_tree() error

### ISSUE THREAD

**Issue by hickop** [godotengine/godot#14975]
**Godot version:**
3.0 5463b5e3489ab2c43279447b66d22b4b35f92e9b

**OS/device including version:**
linux x64

**Issue description:**
Simply adding a Camera outputs in debug:
```
ERROR: get_global_transform: Condition ' !is_inside_tree() ' is true. returned: Transform()
   At: scene/3d/spatial.cpp:265.
ERROR: get_global_transform: Condition ' !is_inside_tree() ' is true. returned: Transform()
   At: scene/3d/spatial.cpp:265.
```

**Steps to reproduce:**
- Create Spatial
- Add Camera
- Run

**Comment by AlmightyScientist** [comment]
The problem is `_set_tree` isn't called for Camera.

I'm on it.

**Comment by SIsilicon** [comment]
I get that bug too on my
Windows 7 64-bit.

**Comment by volzhs** [comment]
pushed PR #14977 for this issue.

**Comment by reduz** [comment]
fixed via #14977
